lib/semantics/check-omp-structure.h

```diff
@@ -72,39 +108,41 @@ class OmpStructureChecker : public virtual BaseChecker {
   void Enter(const parser::OmpReductionClause &);
   void Enter(const parser::OmpScheduleClause &);
 
-  void EmptyAllowed() { allowedClauses_ = {}; }
-  void SetAllowed(std::set<std::type_index> allowedClauses) {
-    allowedClauses_ = allowedClauses;
+  bool HasInvalidCloselyNestedRegion(
+      const parser::CharBlock &, const OmpDirectiveEnumType &);
+  void CheckAllowed(const OmpClauseEnum &);
+
+private:
+  struct OmpContext {
```

Comment:
For private names like this you don't really need the `Omp` prefix. Everything is OMP-related.
So consider just `struct Context`, `GetContext()`, `context_`, etc.

Reply:
Addressed in commit [cb89744](cb89744746186bcf120d8ca62a791e2778366b9b).